Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -1153,10 +1154,13 @@ void CFrame::DoStop()
       }
     }
 
-    if (SConfig::GetInstance().bWii && !m_tried_graceful_shutdown)
+    if (!m_tried_graceful_shutdown && SConfig::GetInstance().bWii &&
+        WII_IPC_HLE_Interface::GetDeviceByName("/dev/stm/eventhook")->IsOpened())
```

Comment:
That can't happen, because Dolphin always initialises the IPC HLE when SConfig::GetInstance().bWii, and the STM device is always added to the device map. So GetDeviceByName should never return nullptr here.